addons/l10n_dz/report/invoice_report.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="invoice_report_amount_in_words_inherit" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[hasclass('clearfix')]" position="after">
+            <div style="padding-top:30px; padding-bottom:20px;">
+                <p>
+                    <h6>
+                        <strong>Arranged the present invoice in the amount of :<span t-esc="o.get_amount2words()"/></strong>
+                    </h6>
+                </p>
+            </div>
```

Comment:
```suggestion
            <div style="padding-top:30px; padding-bottom:20px;">
                <p class="font-weight-bold">
                    Arranged the present invoice in the amount of :<t t-esc="o.get_amount2words()"/>
                </p>
            </div>
```

Are sure about the `padding`? If you want to have space between this `div` and another one, you should use `margin`